include/cantera/oneD/MultiNewton.h

```diff
@@ -45,11 +45,64 @@ class MultiNewton
                      const OneDim& r, int loglevel);
 
     /**
-     * On entry, step0 must contain an undamped Newton step for the solution x0.
-     * This method attempts to find a damping coefficient such that the next
-     * undamped step would have a norm smaller than that of step0. If
-     * successful, the new solution after taking the damped step is returned in
-     * x1, and the undamped step at x1 is returned in step1.
+     * Performs a damped Newton step to solve the system of nonlinear equations.
+     *
+     * On entry, `step0` must contain an undamped Newton step for the solution `x0`.
+     * This method attempts to find a damping coefficient `alpha_k` such that the next
+     * undamped step would have a norm smaller than that of `step0`. If successful,
+     * the new solution after taking the damped step is returned in `x1`, and the
+     * undamped step at `x1` is returned in `step1`.
+     *
+     * This uses the method outlined in Kee et al. @cite kee2003.
+     *
+     * The system of equations can be written in the form:
+     * @f[
+     *   F(x) = 0
+     * @f]
+     *
+     * Where `F` is the system of nonlinear equations, `x` is the solution vector.
+     *
+     * For the damped Newton method we are solving:
+     *
+     * @f[
+     *   x_{k+1} - x_k = \Delta x_k = -\alpha_k J^(-1)(x_k) F(x_k)
+     * @f]
+     *
+     * Where @f$ J @f$ is the Jacobian matrix of @f$ F @f$ with respect to @f$ x @f$, and @f$ \alpha_k @f$ is
+     * the damping factor, and @f$ \Delta x_k @f$ is the Newton step at @fx_k @f, sometimes
```

Comment:
Please check for warnings from Doxygen.
```suggestion
     * the damping factor, and @f$ \Delta x_k @f$ is the Newton step at @f x_k @f, sometimes
```